<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tradeDialog</class>
 <widget class="QDialog" name="tradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>BlankCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#tradeDialog{ 
border-image: url(:/images/trade2.png);
}</string>
  </property>
  <widget class="QLabel" name="debitNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;付 款&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="balanceNameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 16pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;余 额&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="debitLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>65</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 48pt &quot;Swis721 Blk BT&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>2.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>145</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 48pt &quot;Swis721 Blk BT&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>120.00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
